packages/client/src/utils/generateInFolder.ts

```diff
@@ -70,7 +74,17 @@ export async function generateInFolder({
         overwrite: true,
       })
     } else {
-      await getPackedPackage('@prisma/client', outputDir)
+      try {
+        await getPackedPackage('@prisma/client', outputDir)
+      } catch (err) {
+        try {
+          // Fallback to packages/client/package.json
+          await getPackedPackage('@prisma/client', outputDir, path.join(__dirname, '..', '..'))
```

Comment:
This might be a candidate for the Windows failing test. This uses `path.join` which probably uses a OS depending separator, but this really is a "package path" which possibly should always use `/` or something like that?

Reply:
oh interesting, I'll give that a shot.